src/components/HighlightedMarkdown/H.js

```diff
@@ -16,12 +24,12 @@ export default class H extends PureComponent {
     children: PropTypes.node.isRequired
   }
 
-  onClick = () => {
+  onClick() {
     // We need to set the hash manually because <Link/> will not.
     // Also we can't use `onSetActive` callback because it doesn't work if
     // an element is on very bottom so that there is no scroll possible.
     setTimeout(() => {
-      location.hash = this.props.children[0].props.to
+      location.hash = this.to
```

Comment:
@kof you right. It is very implict. But after switching to preact there is no "hidden" change of `this.props.children` - it remains the same as passed from parent component, e.g. - just a string.